packages/frontend/src/components/EditorSettings/EditorDrawer.tsx

```diff
@@ -0,0 +1,57 @@
+import { BiMenuAltLeft } from 'react-icons/bi'
+import {
+  Drawer,
+  DrawerBody,
+  DrawerCloseButton,
+  DrawerContent,
+  DrawerOverlay,
+  HStack,
+  Text,
+} from '@chakra-ui/react'
+import { IconButton } from '@opengovsg/design-system-react'
+
+import EditorSidebar from './EditorSidebar'
+import { DrawerLink } from '.'
+
+interface EditorDrawerProps {
+  links: DrawerLink[]
+  isDrawerOpen: boolean
+  openDrawer: () => void
+  closeDrawer: () => void
+}
+
+export default function EditorDrawer(props: EditorDrawerProps) {
```

Comment:
Like the sidebar component - would prefer we refactor and re-use the existing `NavigationDrawer` component unless there's a big blocker

Reply:
same reason as above